<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="Login_page">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>741</width>
      <height>421</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 255, 255, 255), stop:1 rgba(255, 255, 255, 255));</string>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLineEdit" name="lineEdit_username">
      <property name="geometry">
       <rect>
        <x>122</x>
        <y>30</y>
        <width>141</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_password">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>70</y>
        <width>141</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="PB_Login">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>120</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Login</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="leIP">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>170</y>
        <width>141</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lePort">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>200</y>
        <width>141</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QPushButton" name="UserGetAccountNo_PB">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>30</y>
        <width>151</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Get Account Number</string>
      </property>
     </widget>
     <widget class="QPushButton" name="UserAccountBalance_PB">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>30</y>
        <width>151</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>View Account Balance</string>
      </property>
     </widget>
     <widget class="QPushButton" name="UserViewTransaction_PB_2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>120</y>
        <width>151</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>View Account Transaction</string>
      </property>
     </widget>
     <widget class="QPushButton" name="UserMakeTransaction_PB">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>120</y>
        <width>151</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Make Transaction</string>
      </property>
     </widget>
     <widget class="QPushButton" name="UserTransferAmount_PB">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>230</y>
        <width>151</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Transfer Amount</string>
      </property>
     </widget>
     <widget class="QPushButton" name="UserLogout_PB">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>230</y>
        <width>151</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Log out</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
